src/electron/views/report/unified-header-section.tsx

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import { androidAppTitle } from 'content/strings/application';
+import { BrandWhite } from 'icons/brand/white/brand-white';
+import * as React from 'react';
+
+export const UnifiedHeaderSection = NamedFC('UnifiedHeaderSection', () => {
+    return (
+        <header>
+            <div className="report-header-bar">
```

Comment:
A couple things here:
1. I'd try to use css modules for all styles here.
2. I would also remove the ms-* styles as we should be clear about what styles we're using and why.